src/main/java/frc/team3602/robot/subsystems/ShooterSubsystem.java

```diff
@@ -236,6 +256,9 @@ public void periodic() {
     topVolts = (topShooterMotor.getAppliedOutput() / 12);
     bottomVolts = (bottomShooterMotor.getAppliedOutput() / 12);
 
+    topFeed = (topShooterMotor.getBusVoltage());
```

Comment:
`getBusVoltage()` gets the voltage feeding into the motor controller, not the voltage that is being output to the motor, not sure if that is intentional.